### PR TITLE
Spaces and tabs are the same

### PR DESCRIPTION
**Pull request by atauveron**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ This repository is a personal compilation of coding best practices I learned or
 1. In comments and other forms of documentation, infinitive form ( _e.g._ "add" ), third-person singular simple present ( _e.g._ "adds" ), present participle ( _e.g._ "adding" ) and simple past ( _e.g._ "added" ) can be mixed up
 1. `"` and `'` are the same things, one can use them interchangeably in the same file
 1. `"        "` and `"	"` are the same things, one can also use them interchangeably in the same file
```
Comment (badouralix) [inline README.md]: This line is exactly the same ( first is spaces, second is tabs ), but it is poorly rendered, we should probably get rid of it

**Review by badouralix — COMMENTED** [review]
That is definitely a good one 😂 
Thanks for the contribution

File: README.md (hunk 1 above)
Comment (atauveron) [reply README.md]: Done in commit c52a9f3eb5b20df28ffdfeffaf0479bc3b95974a

**Review by badouralix — APPROVED** [review]
:shipit:

**Pull request merged**